### ISSUE TITLE
Accessibility - Information Icon is Missing ALT Text

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Ilab-do-good#21]
The information icon is missing ALT text to explain what it's function is.

**Comment by adriennerlong** [comment]
I've flagged this as a high priority issue. We will ask the developers to address before 9/11.

**Comment by jfc3-dol** [comment]
@adriennerlong,

Thanks, for getting the developers to fix this item. It's important for those using screen readers.

**Comment by jfc3-dol** [comment]
Thanks, for telling VoiceOver user that it is an information button but it should be a link since it goes to another screen within the application and is not function.

Leaving this issue open.

**Comment by adriennerlong** [comment]
John, we want the information button to link to the About this App page. If ICF changes the information button from a button to a link will that address this issue?

**Comment by jfc3-dol** [comment]
@adriennerlong,

Yes, making it a link will better for the screen reader user. It tells them they should be going to another page or somewhere else on the page they are on.

**Comment by adriennerlong** [comment]
Ok, thanks. ICF, please make the information button a link to the About this App page for screen reader users

**Comment by jfc3-dol** [comment]
@adriennerlong,

VopiceOver now reads the info icon as "info link" but does not allow a VoiceOver user to tap to open the link to go to the information content screen. I can tap on the line with VoiceOver off, and it goes to the screen.

**Comment by jfc3-dol** [comment]
The information link at the bottom on the home screen works with our Android device and TalkBack.

**Comment by adriennerlong** [comment]
From ICF: Initial issue is fixed, button doesn't work.

**Comment by jfc3-dol** [comment]
VoiceOver is now able to open the information LINK.

Closing this issue.
